src/options.js

```diff
@@ -47,6 +47,8 @@ const schema = require('./options.json');
  */
 function getOptions(pluginOptions) {
   const options = {
+    cache: true,
+    cacheLocation: 'node_modules/.cache/.eslintcache',
```

Comment:
The `eslint` places the file in the root of the project.
I think it would be more convenient to put it inside `node_modules`, so there's no need to add it to `.gitignore`